auditor/src/client/mod.rs

```diff
@@ -29,61 +31,130 @@ impl std::fmt::Display for ClientError {
             "{}",
             match self {
                 ClientError::RecordExists => ERR_RECORD_EXISTS.to_string(),
+                ClientError::InvalidTimeout => ERR_INVALID_TIMEOUT.to_string(),
                 ClientError::ReqwestError(e) => format!("Reqwest Error: {e}"),
             }
         )
     }
 }
 
+/// The [`AuditorClientBuilder`] is used to build an instance of
+/// either [`AuditorClient`] or [`AuditorClientBlocking`].
+///
+/// # Examples
+///
+/// Using the `address` and `port` of the Auditor instance:
+///
+/// ```
+/// use auditor::client::AuditorClientBuilder;
+///
```

Comment:
I prefer imports to be hidden in the docs, something like this:

```rust
/// # use auditor::client::AuditorClientBuilder;
/// # 
```

(note that this also excludes the newline to avoid that it is printed)

Reply:
Done!